src/main/kotlin/ru/buhinder/alcopartyservice/dto/response/SearchEventResponse.kt

```diff
@@ -0,0 +1,9 @@
+package ru.buhinder.alcopartyservice.dto.response
+
+import java.util.*
```

Comment:
wildcard import

Reply:
there's 2 data classes for multiple(getAll) and single(getById) responses. Could you please use them instead

> data class MultipleEventResponse(

    val event: EventResponse,
    val images: List<UUID>,
    val isParticipant: Boolean,
)

> data class SingleEventResponse(
    val event: EventResponse,
    val images: List<UUID>, <-- needs to be changed
    val participants: List<UUID>,
)